web/src/lib/components/shared-components/search-bar/search-people-section.svelte

```diff
@@ -38,15 +40,23 @@
     }
     selectedPeople = selectedPeople;
   }
+
+  const filterPeople = (list: PersonResponseDto[], name: string) => {
+    return name == '' ? list : list.filter((p) => p.name.toLowerCase().startsWith(name.toLowerCase()));
```

Comment:
You can move `name.toLowerCase()` to a variable to only evaluate it once.

Reply:
Try to use strict equality by default, in this case you could also do `name ? list : ...`